app/pb_kits/playbook/pb_checkbox/_checkbox.scss

```diff
@@ -0,0 +1,66 @@
+@import "../tokens/colors";
+$transition: 0.2s cubic-bezier(0.95, 0.05, 0.795, 0.035);
+
+[class^=pb_checkbox_kit] {
+  .pb_checkbox_label {
+    display: block;
+    position: relative;
+    padding-left: 30px;
```

Comment:
Could this use a spacing token? - [Spacing Tokens](https://github.com/powerhome/playbook/blob/master/app/pb_kits/playbook/tokens/_spacing.scss#L7)